clustered/server/src/main/java/org/ehcache/clustered/server/store/ClusterTierActiveEntity.java

```diff
@@ -507,8 +517,19 @@ private void invalidateHashForClient(ClientDescriptor originatingClientDescripto
     }
   }
 
+  private void trackMessage(EhcacheOperationMessage message) {
+    ClientMessageTracker clientMessageTracker = stateService.getClientMessageTracker(storeIdentifier);
```

Comment:
This part of code implies that an Active will always be tracking clients, which is not required. I would suggest to not do that. Instead we can push a state in `DefaultClientMessageTracker` during `loadExisting` that its promoted to Active from passive and should not track now.